public/dorequest.php

```diff
@@ -230,7 +230,7 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
      */
 
     case "ping":
-        if ($user === null) {
+        if ($user === null || !$gameID) {
```

Comment:
Verified that the log seems to be associated with pings for game 0 (wonder what's doing that), but the error could still occur if an unknown ID was provided. The exception just indicates `Game::find` returned null.

Reply:
An ID for a game that doesn't exist is fine I'd say. 0 looks like invalid input - it's just a quick fix, better validation will come when we migrate dorequest to controllers?